packages/vitest/package.json

```diff
@@ -21,8 +22,14 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.mjs"
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      },
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
```

Comment:
Note: I had to alter this a bit as the eslint config sorts the `types` key above `require`.

Reply:
The rule comes from https://github.com/antfu/eslint-config/blob/8ddaee3bf8a942b4bc73cd009daf6b67f155c1ad/packages/basic/index.js#L125-L130